i18n/es.json

```
@@ -8006,5 +8006,21 @@
   {
     "id": "api.admin.saml.failure_reset_authdata_to_email.app_error",
     "translation": "No se ha podido restablecer el campo AuthData a Email."
+  },
+  {
+    "id": "api.command_channel_header.update_channel.max_length",
+    "translation": "El texto introducido supera el límite de caracteres. La cabecera del canal está limitada a {{.MaxLength} caracteres."
```

It seems the bot sent a string with an invalid templating substitution:
```suggestion
    "translation": "El texto introducido supera el límite de caracteres. La cabecera del canal está limitada a {.MaxLength} caracteres."
```

---

Fixed in weblate :+1: